src/components/patterns/pill-card.tsx

```diff
@@ -66,7 +66,7 @@ const PillCard = ({imageUrl, imageAlt, videoUrl, isArticle, children, bgColor, .
 
       <div
         className={twMerge(
-          "flex flex-col px-10 pb-[125px] pt-20 @3xl:pb-[175px] @4xl:pb-[225px] @5xl:pb-[250px] @7xl:pb-[300px] lg:px-14",
+          "rs-px-3 flex flex-col pb-[125px] pt-20 @7xl:rs-px-4 @2xl:pb-[175px] @3xl:pb-[225px] @4xl:pb-[300px]",
```

Comment:
let's order these in increasing order `@2xl... @3xl ...`

Reply:
That is strangely not how the linter wants to order it.

Reply:
Unhappy lint 😞 
![image](https://github.com/user-attachments/assets/aba676c9-306b-477f-8612-14e5c87c8e88)